docs/installation/acme/Configuring_ACME_Realm.md

```diff
@@ -87,7 +87,40 @@ usersDN=ou=people,dc=ca,dc=pki,dc=example,dc=com
 groupsDN=ou=groups,dc=ca,dc=pki,dc=example,dc=com
 ```
 
+## Configuring PosgreSQL Realm
+
+The ACME responder can be configured with a PostgreSQL realm.
+
+First, prepare a database (e.g. acme) and a user (e.g. acme) to access the database.
+Verify the database connection with the following command:
+
+```
+$ psql -h $HOSTNAME -d acme -U acme
+```
+
+A sample PostgreSQL realm configuration is available at
+[/usr/share/pki/acme/realm/postgresql/realm.conf](../../../base/acme/realm/postgresql/realm.conf).
+
+To use the PostgreSQL realm, copy the sample realm.conf into the /etc/pki/pki-tomcat/acme folder,
+or execute the following command to customize some of the parameters:
+
+```
+$ pki-server acme-realm-mod --type postgresql \
```

Comment:
Incase of podman & Openshift deployment, how this will work?
Do we need to update podman &  Openshift doc as well?

Reply:
Good catch! Let me update them as well.

Reply:
Let me update the OpenShift and Podman docs in a separate PR.